lua/mapvote/server/modules/net.lua

```diff
@@ -46,10 +46,11 @@ MapVote.Net.receiveWithMiddleware( "MapVote_ChangeVote", function( _, ply )
     end
 
     MapVote.state.votes[ply:SteamID()] = mapID
-
+    local voteMult = MapVote.config.VoteWeightMultipliers[ply:GetUserGroup()] or 1
```

Comment:
A more central place for this would be nicer, something like MapVote.GetVoteMultiplier( ply ) which then calls a hook instead of usergroup, having a hook would be nice as it's more versatile